### PR TITLE
Module 7 edits

### LINKED ISSUE #22: Module 7
- [x] Generally, look for errors from the auto-translation (there are a couple of odd places where things got copied weirdly)
- [x] Mentions to EdStem and "Marking" (and other EdStem terminology).
- [x] Mentions of `/home` and  `/course/lecture-readings` in the text

### PR DESCRIPTION
**Pull request by LuckyQXW**
- Closes #22
- There are a lot of visualizations in this module, and I am not sure how to write the alt text for some of them especially the binary tree diagrams :/

### CODE REVIEW

File: book_source/source/module-7-geospatial-data/lesson-20-dissolve-join/joins.md [hunk 1]
```diff
@@ -5,19 +5,32 @@
...
 We're going to take a quick step away from geospatial data for the next flew slides, but this concept will be very relevant for the example we will be doing with geospatial data.  
 
 Let's go back to the world of CSV data that we would process with `pandas` . Suppose I had two different `DataFrame` shown below. The left one `tas` stores information about the TAs and `grading` on the right stores information about students and which TA will be grading them ( `ta_id` and `grader_id` will match if that TA is grading that student). Our final goal will be trying to find the number of students each TA will grade.  
 
-```{image} https://static.us.edusercontent.com/files/XefeN513eek9ajoDicHmXplz
-:alt: TODO
-:width: 758
-:align: center
-```
+## `tas`
```
Comment (hschafer) [inline book_source/source/module-7-geospatial-data/lesson-20-dissolve-join/joins.md:14]: Minor: Do you think we can get these tables to show up side-by-side? There might be a way to hack this with the [sidebar directive](https://jupyterbook.org/content/layout.html#full-width-markdown-content) or if we include some CSS to make them appear on the same line.

File: book_source/source/module-7-geospatial-data/lesson-20-dissolve-join/hurricane-florence/HurricanFlorence.ipynb [hunk 2]
```diff
@@ -1 +1 @@
...
...pgghYFkWwjCs3GSBqOr4mumJoghZlqFW662wnWadGSklut0uPM9DmqZ7NhurSmzbhqYJfOa6ewAAS90U1/zeBfilX/wk3p3FAACZxXjxSQ/iL9/0I5AAtJUgSVH0h3PWO6aOB1OSrEAhJRxT7/1F9P66VcBFBYf69yb7bk6i140sSTNkaQzb83pBxThCGG99/9X7LBERERFtbSdVEd8C4ONCiFcBOALgJeMZ0nyQUsIwDNbEIKJtUctG4ziuRPag6ugVBAGSJKnEmIBqF16eFNfzoBkm/unqI0hWijqf86aLAQDde06AMGzILIYwbHz8nhPwmd/6LADANjTc/MbnotlcWrPPffv2ATj+PKugyn/ftYRLbzyK3/yRxwMA0rxAe7m57bFrmgbTNJGmKYIgQJYmcF0XWZaNlOVX1aWHRERERFU0UuBHSnkZgMtWfn4AwLPGP6T5sBcnI0Q0XlU7jyRJgqIo4Pt+JZez7gV+UMNimOGVf/sV3HD32ho43pnnYv9Fr1+3xo9n6QPbuq4L6CZ0TeDBToL/uuUYnnTqAk49odci/r2Xfwfv+PzNiLMCX7jxPlzyq8+EqWs7WvLXaDQAAN1ud+DY1nV9o6usEQQBdF3nMi8iIiKiIbEP6gRxOQQRbVdRFGi32/B9H5ZlodncfpbFOGVZVo6r0Wig2+1OJbNxr9R3cT0PhmFCSiCXEjff18LPfuha3Lu8cXaMd+a56xZ1NnStbPvuBzVYpoFPfeMupLnEb3/ivxFnx4tE/+qzH4Nf/MEz8fyzT8YvfvTruOaORTzmd/4D337Tc2Ga1siBnyAIBlqv53leduRK0xSmaaJWq6HT6aAo1m9NL4SA67os6kxEREQ0IgZ+JqhK39ZXaSyTwokAzZs0TRGGITzPg2EYlQkm53mO5eVlWJYF3/cRRdGuF+P2fb8MHkzaNM4tuq5Ds1w4tonXfPhaPP30E/Hp6+7GN767hDTf3rncNjQYugY3qCMtJF7wF5fjxnvW70D54oOnoB1l+MShO/GhVz0Vn7n+HvzRp7+FpW6CwDawb98+JGmGTnvzDpaa1ssQMk0TQC+gqepXWZYFTdPQ6XTKwFCj0UAURUiSpCwqDgCmacL3fQA81xMRERGNioGfCbJte093JiLaLk7sjlMt3Wu1WjlpjuMhqt/ugiRJIKWE53m7fq5TmSK7RS1tM01z
```
Comment (hschafer) [inline book_source/source/module-7-geospatial-data/lesson-20-dissolve-join/hurricane-florence/HurricanFlorence.ipynb:1]: Do we use `data.zip` in this slide? Same for Lesson 19 Hurricane Florence. Also, Lesson 19 is missing the stormhistory.csv from the data files download

**Review by hschafer — APPROVED** [review]
I think the alt texts look fine! It's generally really hard to write good alt texts, so what you have seems good enough!

All of this looks good to me! I only found one bug (data downloads) and then have a few misc feature requests that might make the reading a little bit easier.

File: book_source/source/module-7-geospatial-data/lesson-20-dissolve-join/joins.md (hunk 1 above)
Comment (LuckyQXW) [reply book_source/source/module-7-geospatial-data/lesson-20-dissolve-join/joins.md]: Added some CSS to make them appear on the same slide on wider screen sizes. Let me know if it looks okay (including the ones on the quiz slide)

File: book_source/source/module-7-geospatial-data/lesson-19-geospatial-data-misc-python/matplotlib-plotting.md [hunk 3]
```diff
@@ -108,14 +110,14 @@ This is an example of a **2d-array** (or a `list` of `list` s). This is actually
 For example, you will be making a plot like the following on your next take-home assignment.  
 
 ```{image} https://static.us.edusercontent.com/files/azWntPY0ftQPaBXP5doNIEkU
-:alt: TODO
+:alt: Low food access output from take-home assessment 5, displaying four different types of low access in a two by two grid
```
[the remark this answers is not in the document]
Comment (LuckyQXW) [reply book_source/source/module-7-geospatial-data/lesson-19-geospatial-data-misc-python/matplotlib-plotting.md:113]: I made it full width, but the original image has white padding around the plots and that could be another reason why it looks small.

**Comment by LuckyQXW** [comment]
I have fixed some merge conflicts- it's probably a good idea to double check that I get the changes form main properly.

**Comment by hschafer** [comment]
Styling changes look good! 

I think the merge I made with changes to #52 got messed up here somehow. Not your fault, since I didn't think that touches the changes you made it got complicated since you renamed one of the lessons in this commit. I'm going to push a commit to this branch with the fixed images and remade zips, and then I'll merge it!

**Comment by LuckyQXW** [comment]
Sure, but I have also remade the zips when I was fixing the merge conflicts so that might not be necessary.

**Pull request merged**